bugz.py

```diff
@@ -124,20 +124,26 @@ def bugzilla_qe_verify(self):
 
                     rows.append(row)
 
-        # Create the DataFrame
-        df = pd.DataFrame(rows)
+        self.db.qa_needed_delete()
 
-        df['modification_date'] = pd.to_datetime(df['modification_date'], format='%Y%m%dT%H:%M:%S') # noqa
-        df['creation_date'] = pd.to_datetime(df['creation_date'], format='%Y%m%dT%H:%M:%S') # noqa
+        if not rows:
+            print("There are no bugs to verify today")
```

Comment:
I assume this is diagnostic for the github action logs 👍🏼

Reply:
yes, to know why did not have data if that's the case and not due to any other error / situation ..